guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -2,9 +2,43 @@
 = Managing Alternate Content Sources
 
 Alternate content sources define alternate paths to download content during synchronization.
-The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
```

Comment:
```suggestion
The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
Alternate content source paths consist of a base URL and any number of appended subpaths. If your base URL is https://server.example.com and your subpaths are rhel7/ and rhel8/, then both https://server.example.com/rhel7/ and https://server.example.com/rhel8/ will be searched.
```

Reply:
> Alternate content source paths consist of a base URL and any number of appended subpaths. If your base URL is https://server.example.com and your subpaths are rhel7/ and rhel8/, then both https://server.example.com/rhel7/ and https://server.example.com/rhel8/ will be searched.

@ianballou does this apply to both RHUI and custom ACS?

Reply:
It does indeed.

Reply:
Hey @ianballou I added this information to the RHUI and Custom ACS prerequisites.